cmd/fuzzcrypto/std/rsa_test.go

```diff
@@ -123,11 +123,14 @@ func FuzzRSASignPSS(f *testing.F) {
 		hashed := sha256.Sum256(msg)
 		sig, err := rsa.SignPSS(rand.Reader, key, crypto.SHA256, hashed[:], &opts)
 		if err != nil {
-			switch err.Error() {
 			// Key size being too small is an invalid input, not a problem.
+			switch err.Error() {
 			case "crypto/rsa: key size too small for PSS signature", "crypto/rsa: invalid key size":
```

Comment:
I take it the idea here is to leave `"crypto/rsa: key size too small for PSS signature"` in place for back compat even though it's a typed error now?

Reply:
Yep, just in case we want to fuzz an older Go version.